scripts/my_local_dev_vars.example

```diff
@@ -11,8 +11,8 @@ OPENSHIFT_SERVER_PORT=8443
 # BROKER_IP_ADDR must be the IP address of where to reach broker
 #   it should not be 127.0.0.1, needs to be an address the pods will be able to reach
 BROKER_IP_ADDR=${OPENSHIFT_SERVER_HOST}
-DOCKERHUB_USERNAME=""
-DOCKERHUB_PASSWORD=""
+DOCKERHUB_USERNAME="changeme"
+DOCKERHUB_PASSWORD="changeme"
```

Comment:
Can we comment on this? I seem to remember that @jwmatthews mentioned this was a valid account and not meant to be a reminder to folks to change them. If I see this I think I *HAVE* to change them. Unless that is what we are going for then leave it as is.

Reply:
it may actually be better to comment them out and default to `"changeme"` where we use it. Does that work?

Reply:
+1 to commenting out and defaulting to changeme. That way it will most likely work and if they want to use their own, they can add them.